src/amo/containers/CategoryPage.js

```diff
@@ -0,0 +1,41 @@
+import { connect } from 'react-redux';
+import { asyncConnect } from 'redux-connect';
+import { compose } from 'redux';
+
+import SearchPage from 'amo/components/SearchPage';
+import {
+  filtersMatch,
+  loadCategoryResultsIfNeeded,
+  parsePage,
+} from 'core/searchUtils';
+
+
+export function mapStateToProps(state, ownProps) {
+  const filters = {
```

Comment:
But these filters are declared here to compare them with state and make sure everything matches. They're passed to `if (filtersMatch([...])) {`, so moving them into there would mean duplication.

They're only passed to the state if they match.